Source/Particles/Collision/BinaryCollision/NuclearFusionFunc.H

```diff
@@ -192,6 +209,13 @@ public:
                     m_probability_threshold,
                     m_probability_target_value,
                     m_fusion_type, engine);
+
+#if (defined WARPX_DIM_RZ)
+          amrex::ParticleReal const u1xbuf_new = u1x[I1[i1]];
```

Comment:
@Yin-YinjianZhao Could you re-indent this part of the code, so that it agrees with the rest of the code?